### ISSUE TITLE
no eq instance?

### ISSUE THREAD

**Issue by mjhoy** [purescript-contrib/purescript-js-date#14]
Just curious, I was wondering why this is the case?

I imagined something like,

```javascript
exports.eqJSDate = function (d1) {
  return function (d2) {
    return d1.getTime() == d2.getTime();
  };
};
```

but perhaps there are issues with that?

**Comment by garyb** [comment]
Yeah, something like that would work - it doesn't need to be implemented in the FFI since those functions already exist in PS.

I didn't add any instances for `JSDate` as I just saw it as a temporary thing that would be used temporarily before conversion to/from the non-JS-specific `DateTime` type instead, but it seems that was wishful thinking. :wink:

**Comment by mjhoy** [comment]
ah, cool. would you accept a pr? i'll try to find those functions...

as an aside, we are finding that working with JSDate objects often makes the most sense and makes the code simpler, especially for front end work that uses react components.

**Comment by thomashoneyman** [comment]
This was resolved by #17
